clients/core-catalog-writer/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
+    id("org.openapi.generator") version "7.3.0"
+}
+val currentDirectory = "$rootDir/clients/core-catalog-writer"
+val outputDirectory: String = project(":libs:core-catalog-writer").projectDir.path
+openApiGenerate {
+    generatorName.set("kotlin-spring")
+    inputSpec.set("$currentDirectory/spec/api.yaml")
+    configOptions.putAll(
+        mapOf(
+            "useSpringBoot3" to "true",
+            "exceptionHandler" to "false",
+            "useSwaggerUI" to "false",
+            "documentationProvider" to "none",
+            "basePackage" to "openBook",
+            "apiPackage" to "openBook.api",
+            "modelPackage" to "openBook.model",
+            "apiSuffix" to "Api",
+            "skipGeneratePom" to "true",
+            "artifactId" to "core-openBook",
+            "useBeanValidation" to "false",
+            "interfaceOnly" to "true",
+            "skipDefaultInterface" to "true",
+            "useTags" to "true",
+            "skipDefaultInterface" to "true",
+            "reactive" to "true"
+        )
+    )
+    generateApiDocumentation.set(false)
+    generateModelDocumentation.set(false)
+    generateApiTests.set(false)
+    generateModelTests.set(false)
+    outputDir.set(outputDirectory)
```

Comment:
As far as I know, in a Kotlin DSL you can write `variable = value`, which is more convenient to read.